internal/cobraext/const.go

```diff
@@ -27,6 +27,9 @@ const (
 	FailFastFlagName        = "fail-fast"
 	FailFastFlagDescription = "fail immediately if any file requires updates (do not overwrite)"
 
+	ForkFlagName        = "fork"
+	ForkFlagDescription = "use fork mode"
```

Comment:
This description is rather vague. I'm okay with it if we only really expect CI to use this flag. But if there is any potential for human users to use it, it might be nice to explain this flag a bit more here.

Reply:
Fixed